### PR TITLE
Adding docs!

### PR DESCRIPTION
**Pull request by Morantron**
docs docs docs :memo:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,15 +69,21 @@ syncCollections({todos: Todos}, store);
...
+* `action creator` will be constructed with 4 possible actions: `add`, `merge`,
+  `remove`, and `reset`.
+* Special `ear` object will be set up to listen to all collection events and
+  trigger right actions depending on the event type.
 * Reducer will be registered in the store under `todos` key.
-* All previous reducers in your store will be replaced.  
+* All previous reducers in your store will be replaced.
 
-You are done. Now any change to `Todos` collection  will be reflected in the redux store.
+You are done. Now any change to `Todos` collection  will be reflected in the
```
Comment (mllocs) [inline README.md:81]: `Todos` is an instance or a class?

Comment (somebody32) [inline README.md:81]: instance

File: README.md [hunk 2]
```diff
@@ -301,6 +233,209 @@ export default function() {
...
+If you don't need a custom serializer you can use:
+
+```javascript
+// keys are reducer names, and values are backbone collections
+const collectionMap = {
+  reducer_name: collection
+}
+```
+
+If you can add extra configuration specifying `serializer` and `indexes_map` keys.
```
Comment (somebody32) [inline README.md]: If you want?

File: README.md [hunk 3]
```diff
@@ -301,6 +233,209 @@ export default function() {
...
+    indexes_map: indexes_map
+  }
+}
+```
+
+#### indexesMap <a id="indexes-map"></a>
+
+With `indexesMap` you can specify the way your entities are indexed in the tree.
+
+`fields` lets you access a *single* entity by a field ( for example `id`, `email`, etc ).
```
Comment (somebody32) [inline README.md]: no need for spaces in ()

File: README.md [hunk 4]
```diff
@@ -301,6 +233,209 @@ export default function() {
...
+        {id: 2, name: 'Mark', org_id: 2, token: '002', __optimistic_id: 'c02'}
+      ]
+    }
+  }
+  */
+```
+
+And to remove indexes at all, just pass empty object as `indexes_map` for `syncCollections`.
+
+#### serializer
```
Comment (somebody32) [inline README.md]: Capital here?

**Comment by somebody32** [comment]
Great :clap: :postal_horn: & :+1:

**Pull request merged**